utils/uri-builder.js

```diff
@@ -1,64 +1,37 @@
 const appRoot = require('app-root-path');
 const config = require('config');
-const decodeUriComponent = require('decode-uri-component');
+const queryString = require('query-string');
 const url = require('url');
 
 const { openapi: { basePath } } = appRoot.require('utils/load-openapi');
 
 const { protocol, hostname } = config.get('server');
 
 /**
- * @summary Self-link builder
- * @function
- * @param {string} id resource ID
- * @param {string} resourcePath resource path
- * @returns A self-link URL
+ * @summary API base URL
  */
-const idSelfLink = (id, resourcePath) => url.format({
-  protocol,
-  hostname,
-  pathname: `${basePath}/${resourcePath}/${id}`,
-});
+const apiBaseURL = url.format({ protocol, hostname, pathname: basePath });
 
 /**
- * @summary Top level query link builder
+ * @summary Resource path link builder
  * @function
- * @param {object} query
+ * @param {string} baseURL base URL
```

Comment:
From our discussion on naming camel case variables with acronyms in them, shouldn't this be called `baseUrl` instead of `baseURL`?

Reply:
Good catch.